tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/AudioOps.java

```diff
@@ -144,4 +147,11 @@ public Mfcc mfcc(Operand<TFloat32> spectrogram, Operand<TInt32> sampleRate,
       Mfcc.Options... options) {
     return Mfcc.create(scope, spectrogram, sampleRate, options);
   }
+
+  /**
+   * Get the parent {@link Ops} object.
+   */
+  public final Ops ops() {
+    return ops;
+  }
```

Comment:
Do we need this to be public?

Reply:
Unless you'd rather expose the field.  The point, as I understood it, was to allow a user to get to the Ops instance from a subgroup like MathOps, which requires it being public.  I could just make the final field public but the codebase seems to prefer accessor functions as is typical for Java.

Reply:
Ah I forgot that when adding an extension method to `MathOps`, you are not in the scope of its package but in the public scope. So yes, this needs to be public then, all good, thanks